docs/frontend.md

```diff
@@ -3,313 +3,186 @@ id: frontend
 title: Frontend
 ---
 
-## React/Redux Support
+This document describes approaches that you can use to implement a frontend for a reSolve application. The following approaches are available:
```

Comment:
I just wanted to make sure that "frontend" is the technically correct (and widely-used) term - and not "front end". Is this correct?

Reply:
This is a common controversy. Both options are used, but we write "frontend" everywhere just for consistency and because "backend" is most often prefered over "back end".

Reply:
Okay, thank you for the clarification.